governance/mixed.go

```diff
@@ -19,7 +21,7 @@ import (
 //                     Note that some items are not backed by defaultParams.
 //
 type MixedEngine struct {
-	initialConfig *params.ChainConfig
+	config *params.ChainConfig
```

Comment:
Please add a comment that inherits handler.go:72, e.g. " Every instances of blockchain, cn, worker, and governance share the chainConfig"

Reply:
done